packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -4,225 +4,242 @@ import type { BadgeState } from './Badge.types';
 
 export const badgeClassName = 'fui-Badge';
 
-const useStyles = makeStyles({
-  root: {
+const useRootStyles = makeStyles({
+  base: {
     display: 'inline-flex',
     boxSizing: 'border-box',
     alignItems: 'center',
     justifyContent: 'center',
-    backgroundColor: tokens.colorBrandBackground,
-    ...shorthands.borderColor(tokens.colorBrandBackground),
-    color: tokens.colorNeutralForegroundOnBrand,
     fontWeight: tokens.fontWeightSemibold,
     ...shorthands.borderWidth(tokens.strokeWidthThin),
     ...shorthands.borderStyle('solid'),
     fontFamily: tokens.fontFamilyBase,
     position: 'relative',
   },
-  rootTiny: {
+
+  // size
+
+  tiny: {
     width: '6px',
     height: '6px',
     fontSize: '4px',
   },
-  rootExtraSmall: {
+  'extra-small': {
     width: '10px',
     height: '10px',
     fontSize: '6px',
   },
-  rootSmall: {
+  small: {
     minWidth: '16px',
     height: '16px',
     ...shorthands.padding('2px'),
     ...shorthands.gap('4px'),
     fontSize: '8px',
   },
-  rootMedium: {
+  medium: {
     height: '20px',
     minWidth: '20px',
     ...shorthands.gap('4px'),
     ...shorthands.padding('4px'),
     fontSize: '10px',
   },
-  rootLarge: {
+  large: {
     minWidth: '24px',
     height: '24px',
     ...shorthands.padding('4px'),
     fontSize: '12px',
     ...shorthands.gap('4px'),
   },
-  rootExtraLarge: {
+  'extra-large': {
     minWidth: '32px',
     height: '32px',
     ...shorthands.padding('6px'),
     ...shorthands.gap('6px'),
     fontSize: '12px',
     ...shorthands.borderWidth(tokens.strokeWidthThick),
   },
-  rootRounded: {
+
+  // shape
+
+  square: {
+    // Default border radius
+  },
+  rounded: {
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
   },
-  rootRoundedSmallToTiny: {
+  roundedSmallToTiny: {
     ...shorthands.borderRadius(tokens.borderRadiusSmall),
   },
-  rootCircular: {
+  circular: {
     ...shorthands.borderRadius('99px'),
   },
-  rootGhost: {
-    backgroundColor: 'transparent',
-    ...shorthands.borderStyle('none'),
-    color: tokens.colorBrandBackground,
+
+  // appearance
+
+  ghost: {
+    ...shorthands.borderColor('transparent'),
```

Comment:
There are 2 consequences of applying a transparent border to `ghost`

1. Ghost badges are wider than before - not too bad since it will have the same width as other badges
2. ghost badges will have a border in windows high contrast mode. This border won't display in the hard coded HC theme we currently have since it's transparent. I'm OK with those being different and AFAIK design are happy with us taking some initiative with windows high contrast mode.

They're not big issues, but just wanted to make it visible

Reply:
1. Yeah, that was the goal of this change, to make ghost the same size as other badges.
2. I didn't realize about high contrast. That is less than ideal. 

I also noticed that actually every badge type _except_ ghost previously had the wrong padding, since the padding specified in the visual spec _includes_ the border width (so, need to subtract the border width from the padding specified in the spec). I think I'll make that change in a follow-up PR, and with that fix, ghost will be the same size as the other badges.

Reply:
For now, I've reverted this ghost change, and have it back to borderStyle: none